src/api/doLoadRegions.js

```
@@ -0,0 +1,7 @@
+import { getCostApi } from './entities.js';
+
+export const doLoadRegions = () =>
+  getCostApi()
+    .listAwsRegions()
+    .then((response) => response.data)
+    .catch(() => {});
```

Do we want to silently fail?